### PR TITLE
Feature/md bv 243

### PR DESCRIPTION
**Pull request by dzeckelev**
added `config` column for `product` table

### CODE REVIEW

File: data/schema.sql [hunk 1]
```diff
@@ -144,7 +144,8 @@ CREATE TABLE products (
     is_server boolean NOT NULL, -- product is defined as server (Agent) or client (Client)
     salt bigint NOT NULL, -- password salt
     password sha3_256 NOT NULL,
-    client_ident client_ident_type NOT NULL
+    client_ident client_ident_type NOT NULL,
+    config json  -- Store configuration of product --
```
Comment (furkhat) [inline data/schema.sql]: not null?

Comment (dzeckelev) [reply data/schema.sql]: @furkhat as I understand the configuration can be downloaded after creating the object in database. Should it not null?

Comment (furkhat) [reply data/schema.sql]: It is basically not null all the time, because postgres json considers null as invalid json. (At least this is what I've seen :)

Comment (dzeckelev) [reply data/schema.sql]: I was guided by the fact that `endpoints.additional_params` and `offerings.additional_params` do not have `NOT NULL`.
`json` fields do not participate in the index, in contrast to `jsonb`, but if you think what you need here I would not mind.
On the other hand, Null values are always not very good.
@ababo what is your opinion?

**Pull request closed without merging**